### PR TITLE
F7-Dark/Light-Mode-Toggle-Redux

### PR DESCRIPTION
**Pull request by AntonioNik**
CurrentNavLightMode
CurrentNavDarkMode

### CODE REVIEW

File: Frontend/src/main.tsx [hunk 1]
```diff
@@ -1,13 +1,17 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
-import { GoogleOAuthProvider } from "@react-oauth/google";
+import { Provider } from "react-redux";
+import { store } from "../src/store/DarkMode/store";
 import App from "./App.tsx";
-import "./index.css";
+import { GoogleOAuthProvider } from "@react-oauth/google";
+import "./main.css";
```
Comment (betinabp) [inline Frontend/src/main.tsx:7]: order the imports

File: Frontend/src/main.css [hunk 2]
```diff
@@ -0,0 +1,11 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+::-webkit-scrollbar {
+  width: 8px;
+}
+
+::-webkit-scrollbar-track {
+  background-color: #f1f1f1;
```
Comment (betinabp) [inline Frontend/src/main.css:10]: set variable for the colors

File: Frontend/src/components/navbar/Navbar.tsx [hunk 3]
```diff
@@ -207,13 +213,24 @@ const Navbar = ({
...
+                  }
+                >
                   {page}
                 </StyledLink>
               </Button>
             ))}
           </Box>
           <Box sx={{ flexGrow: 0 }}>
+            <Tooltip title="Toggle Dark Mode">
+              <span style={{ marginRight: "12px" }}>
```
Comment (betinabp) [inline Frontend/src/components/navbar/Navbar.tsx:230]: Use classes for styling. Avoid using inline styles.

**Review by betinabp — CHANGES_REQUESTED** [review]

File: Frontend/src/components/navbar/Navbar.tsx (hunk 3 above)
Comment (AntonioNik) [reply Frontend/src/components/navbar/Navbar.tsx:230]: in that case when I only need styles for one html tag should I create a scss file and import it to use it or there is a better approach

Comment (betinabp) [reply Frontend/src/components/navbar/Navbar.tsx:230]: You can ceate a scss file. Stick to classes for styling

File: Frontend/src/pages/home/Home.tsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const Home = () => {
```
Comment (betinabp) [inline Frontend/src/pages/home/Home.tsx:3]: Add type for Home. image

**Review by betinabp — CHANGES_REQUESTED** [review]

**Review by betinabp — APPROVED** [review]

**Pull request merged**